velox/exec/RowContainer.cpp

```diff
@@ -159,20 +159,13 @@ RowContainer::RowContainer(
   // A distinct hash table has no aggregates and if the hash table has
   // no nulls, it may be that there are no null flags.
   if (!nullOffsets_.empty()) {
-    // Aggregates start life as null, join dependent columns as non-null.
-    initialNulls_.resize(nullBytes, isJoinBuild_ ? 0x0 : 0xff);
-    // The free flag has an initial value of 0.
-    bits::clearBit(
-        initialNulls_.data(), freeFlagOffset_ - nullOffsets_.front());
-    if (nullableKeys) {
-      for (int32_t i = 0; i < keyTypes_.size(); ++i) {
-        bits::clearBit(initialNulls_.data(), i);
-      }
+    initialNulls_.resize(nullBytes, 0x0);
+    // Aggregates are null on a new row.
+    auto aggregateNullOffset = nullableKeys ? keyTypes.size() : 0;
```

Comment:
This change can be extracted into a separate PR as it seems unrelated to spilling logic.